bridge/client/app/_components/ktb-user/ktb-user.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="user">
+  <p>User : {{user}}</p>
```

Comment:
![image](https://user-images.githubusercontent.com/6098219/115222944-8f8c9780-a10b-11eb-89f5-342777ade07e.png)

Can we make it `User: {{user}}` as it is with `Keptn version: {{version}}` and Bridge `version: {{bridgeVersion}}` ?

Reply:
Thanks for the review. Corrected with the latest commit